docs/developers/publishing.md

```diff
@@ -10,7 +10,7 @@ Note the minimum extension age requirement of 30 days.
 
 ## ESM
 
-If you are utilizing ESM, you probably still want to publish an UMD bundle of your extension. Because Chart.js v3 is tree shakeable, the interface is a bit different.
+If you are utilizing ESM, you probably still want to publish a UMD bundle of your extension. Because Chart.js v3 is tree shakeable, the interface is a bit different.
```

Comment:
Afaik this needs to be an since next word starts with a vowel

Reply:
I'm by no means a grammar expert, so I can't be 100% sure, but as a native speaker "an UMD bundle" doesn't sound right at all to me

Reply:
I think this change is correct, but definitely an exception to the rule